src/Form/Tag/FormTagsManager.php

```diff
@@ -40,14 +40,14 @@
 
 final class FormTagsManager
 {
-    public function getTags(Form $form): array
+    public function getTags(Form $form, string $filter = ""): array
     {
         $tags = [];
         foreach ($this->getTagProviders() as $provider) {
             $tags = array_merge($tags, $provider->getTags($form));
         }
 
-        return $tags;
+        return $this->filterTags($tags, $filter);
```

Comment:
No need to do any operation if filter is empty.
```suggestion
        return $filter === '' ? $tags : $this->filterTags($tags, $filter);
```